test/Microsoft.AspNetCore.Server.Kestrel.FunctionalTests/RequestTests.cs

```diff
@@ -420,6 +422,42 @@ public void CanUpgradeRequestWithConnectionKeepAliveUpgradeHeader()
             }
         }
 
+        [Fact]
+        public async Task RequestAbortedTokenFiredOnClientFIN()
```

Comment:
Do we have a test verifying the RequestAborted token *isn't* fired for the last successful request made before the connection is closed?